### ISSUE TITLE
Latest release on jCenter is 1.0.3

### ISSUE THREAD

**Issue by AlexMeuer** [Estimote/Android-SDK#262]
The most recent release on [jCenter (https://bintray.com/bintray/jcenter/com.estimote:sdk)](https://bintray.com/bintray/jcenter/com.estimote:sdk) is 1.0.3 which is quite a few releases behind. Quite frustrating when you need 1.0.9 or greater to detect beacons on Android N and up (for which there are about 4 or 5 open issues.)
Here's the jCenter badge showing the most recent available version:
[ ![Download](https://api.bintray.com/packages/bintray/jcenter/com.estimote%3Asdk/images/download.svg) ](https://bintray.com/bintray/jcenter/com.estimote%3Asdk/_latestVersion)

**Comment by AlexMeuer** [comment]
From part one of the [Android tutorial](http://developer.estimote.com/android/tutorial/part-1-setting-up/):

> // add the following line, and replace "1.0.12" with the latest version
    // of Estimote Android SDK; you'll find the latest version number on:
    //   https://github.com/Estimote/Android-SDK/releases
    compile 'com.estimote:sdk:1.0.12'

**Comment by pawelDylag** [comment]
Hey!

1. I will investigate this, but our current version is available at JCenter afaik-> http://jcenter.bintray.com/com/estimote/sdk/

2. This SDK is for now only updated with management stuff (connecting to beacons, mesh, etc). 
It's because we did a total overhaul to the scanning mechanism in order to make it work on Android 6.0, 7.0 and 8.0. Because of so many breaking changes, we decided to release it as a separate library here -> https://github.com/Estimote/Android-SDK/tree/proximity-sdk

3. I really encourage you to read and track this topic at our forums. I explained there what we are doing, why, and I keep posting an updates about current progress:
https://forums.estimote.com/t/changes-to-android-sdk-current-progress/7450/9

4. Unfortunately, due to the vast changes in our stack, the developer portal is trailing behind a little bit, as well as documentation or new stuff -> we're working on that!

Cheers!
